package.json

```diff
@@ -219,8 +219,8 @@
     "web3": "^0.20.7",
     "web3-eth-abi": "^1.2.6",
     "web3-stream-provider": "^4.0.0",
-    "web3-utils": "^1.2.4",
-    "xtend": "^4.0.1"
+    "xtend": "^4.0.1",
```

Comment:
I think this must not be commited on this PR right? @JONAF2103

Reply:
We are using web3-utils in some lumino classes, or this will be changed?

Reply:
Yup this is a typo probably

Reply:
This line was added because, after the merge, "xtend" was deleted, but, if it is not necessary I can remove it.

Reply:
Fixed @fmelo  no worries